src/utils/countInputValidation.js

```diff
@@ -0,0 +1,12 @@
+import { ERROR_MESSAGES } from "../constants/messages.js";
+
+const countInputValidation = (COUNT) => {
+  if (isNaN(COUNT)) {
+    throw new Error(ERROR_MESSAGES.COUNT_ERROR_MESSAGES.ONLY_NUMBER);
+  }
+  if (Number(COUNT) < 1) {
+    throw new Error(ERROR_MESSAGES.COUNT_ERROR_MESSAGES.NOT_NEGATIVE);
+  }
```

Comment:
```suggestion
  if (COUNT < 1) {
    throw new Error(ERROR_MESSAGES.COUNT_ERROR_MESSAGES.NOT_NEGATIVE);
  }
```
COUNT가 string이긴 하지만 조건문에 넣으면 암시적으로 형 변환이 이뤄나면서 Number없이 예를 들면 "1"은 string -> number로 자동으로 변환되어서 저 코드는 빼도 좋을듯합니다. 대신 정확한걸 원하신다면 지금을 유지하는것이 더 좋을거같아서 한번 말씀드려봅니다!

Reply:
오 그건 몰랐네요 ! 감사합니다